utils/hct/gen_intrin_main.txt

```diff
@@ -746,44 +726,14 @@ uint [[ro]] Load(in uint byteOffset) : byteaddress_load;
 uint<2> [[ro]] Load2(in uint byteOffset) : byteaddress_load;
 uint<3> [[ro]] Load3(in uint byteOffset) : byteaddress_load;
 uint<4> [[ro]] Load4(in uint byteOffset) : byteaddress_load;
-half [[ro]] LoadHalf(in uint byteOffset) : byteaddress_load;
-half<2> [[ro]] LoadHalf2(in uint byteOffset) : byteaddress_load;
-half<3> [[ro]] LoadHalf3(in uint byteOffset) : byteaddress_load;
-half<4> [[ro]] LoadHalf4(in uint byteOffset) : byteaddress_load;
-float [[ro]] LoadFloat(in uint byteOffset) : byteaddress_load;
-float<2> [[ro]] LoadFloat2(in uint byteOffset) : byteaddress_load;
-float<3> [[ro]] LoadFloat3(in uint byteOffset) : byteaddress_load;
-float<4> [[ro]] LoadFloat4(in uint byteOffset) : byteaddress_load;
-double [[ro]] LoadDouble(in uint byteOffset) : byteaddress_load;
-double<2> [[ro]] LoadDouble2(in uint byteOffset) : byteaddress_load;
 uint [[]] Load(in uint byteOffset, out uint_only status) : byteaddress_load_s;
 uint<2> [[]] Load2(in uint byteOffset, out uint_only status) : byteaddress_load_s;
 uint<3> [[]] Load3(in uint byteOffset, out uint_only status) : byteaddress_load_s;
 uint<4> [[]] Load4(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-half [[]] LoadHalf(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-half<2> [[]] LoadHalf2(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-half<3> [[]] LoadHalf3(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-half<4> [[]] LoadHalf4(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-float [[]] LoadFloat(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-float<2> [[]] LoadFloat2(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-float<3> [[]] LoadFloat3(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-float<4> [[]] LoadFloat4(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-double [[]] LoadDouble(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-double<2> [[]] LoadDouble2(in uint byteOffset, out uint_only status) : byteaddress_load_s;
-void [[]] Store(in uint byteOffset, in uint value) : byteaddress_store;
+void [[]] Store(in uint byteOffset, in numeric<c> value) : byteaddress_store;
```

Comment:
Do we need two template overloads here?  One with `numeric value` and the other with `numeric<c> value` to make it support scalar or vector, but not matrix?
Also, we might need a new identifier for this function to differentiate from 2/3/4 versions.  Same with Load().